pkg/controllers/managementuser/secret/secret.go

```diff
@@ -68,6 +100,135 @@ func Register(ctx context.Context, cluster *config.UserContext) {
 
 		return obj, nil
 	})
+
+	directSyncController := &DirectSyncController{
+		upstreamSecrets:   mgmt.Core.Secrets("fleet-default"),
+		downstreamSecrets: cluster.Core.Secrets(""),
+		clusterName:       clusterRec.Spec.DisplayName,
+	}
+
+	directSyncController.upstreamSecrets.AddHandler(ctx, "secret-direct-synced", directSyncController.sync)
+}
+
+func (dsc *DirectSyncController) bootstrap(mgmtClusterClient v3.ClusterInterface, mgmtCluster *apimgmtv3.Cluster) error {
```

Comment:
```suggestion
func (c *DirectSyncController) bootstrap(mgmtClusterClient v3.ClusterInterface, mgmtCluster *apimgmtv3.Cluster) error {
```
honestly I'd just use a single character name, I keep thinking that `dsc` is a variable when reviewing.